packages/js/settings-editor/changelog/52806-fix-settings-typings

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: tweak
+Comment: Updates unreleased feature
```

Comment:
Nit: I think we need a bit more specific with the changelog here, we've been called out once for having an overly generic description 😅